runtimes/huggingface/tests/test_common.py

```diff
@@ -1,6 +1,8 @@
+from unittest.mock import patch, MagicMock
```

Comment:
:question: Is there a particular order defined for imports?  I can't see grouping by standard library vs. third-party groupings,  so not sure if there is an expectation around this

Reply:
Fixed!

Reply:
I meant for blocks of imports, rather than imports within a single line.  It's okay if not, but wanted to clarify what I meant

Reply:
Also, `black` settings are configured from pyproject.toml in the root folder under `[tool.black]` section. In this case, the line length is not set so it's using a default length of 88 chars.